product_docs/docs/postgres_distributed_for_kubernetes/1/tde.mdx

```diff
@@ -1,45 +1,45 @@
 ---
-title: 'Transparent Data Encryption (TDE)'
+title: 'Transparent data encryption (TDE)'
 originalFilePath: 'src/tde.md'
 ---
 
 !!! Important
     TDE is available *only* for operands that support it:
-    EPAS versions 15 and newer, Postgres Extended versions 15 and newer.
+    EDB Postgres Advanced Server versions 15 and newer and EDB Postgres Extended versions 15 and newer.
 
-Transparent Data Encryption, or TDE, is a technology used by several database
-vendors to **encrypt data at rest**, i.e. database files on disk.
-TDE does not however encrypt data in use.
+Transparent data encryption, or TDE, is a technology used by several database
```

Comment:
Should remain capitalized.

Reply:
Not referring to the product but the technology.